en/docs/publish/api-microgateway/quick-start-with-kubernetes.md

```diff
@@ -0,0 +1,113 @@
+        export PATH=$PATH:<CLI_TOOL_EXTRACTED_LOCATION>
+        ```
+
+        ```bash
+        apictl init petstore --oas https://petstore.swagger.io/v2/swagger.json
+        ```
+
+        The project is now initialized. You should notice a directory with name "petstore" being created in the location 
+        where you executed the command. Let's Zip the created "petstore"` directory and create a Kubernetes condfigmap.
+        You can also use this Kubernetes configmap to deploy APIs.
+
+        ```bash
+        zip -r petstore.zip petstore/
+        kubectl create cm petstore-cm --from-file petstore.zip
+        ```
+
+2.  Now let's deploy our first API by creating an API resource in Kubernetes.
+
+    ```bash
+    cat <<EOF | kubectl apply -f -
+    apiVersion: wso2.com/v1alpha2
+    kind: API
+    metadata:
+        name: petstore-api
+    spec:
+        swaggerConfigMapName: petstore-cm
+    EOF
+    ```
+
+    Or else execute the following command
+
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/wso2/k8s-api-operator/v2.0.0-m3/scenarios/scenario-2/petstore-api.yaml
+    ```
+
+### Step 3 - Invoke the API
+
+1.  The next step would be to invoke the API using a REST tool. Since APIs on the Microgateway are by default secured. 
+    We need a valid token in order to invoke the API. 
+    Use the following sample token accepted by the microgateway to access the API. 
+    Lets set the token to command line as a variable.
+
+    ```bash
+    TOKEN=eyJ4NXQiOiJNell4TW1Ga09HWXdNV0kwWldObU5EY3hOR1l3WW1NNFpUQTNNV0kyTkRBelpHUXpOR00wWkdSbE5qSmtPREZrWkRSaU9URmtNV0ZoTXpVMlpHVmxOZyIsImtpZCI6Ik16WXhNbUZrT0dZd01XSTBaV05tTkRjeE5HWXdZbU00WlRBM01XSTJOREF6WkdRek5HTTBaR1JsTmpKa09ERmtaRFJpT1RGa01XRmhNelUyWkdWbE5nX1JTMjU2IiwiYWxnIjoiUlMyNTYifQ==.eyJhdWQiOiJBT2syNFF6WndRXzYyb2QyNDdXQnVtd0VFZndhIiwic3ViIjoiYWRtaW5AY2FyYm9uLnN1cGVyIiwibmJmIjoxNTk2MDA5NTU2LCJhenAiOiJBT2syNFF6WndRXzYyb2QyNDdXQnVtd0VFZndhIiwic2NvcGUiOiJhbV9hcHBsaWNhdGlvbl9zY29wZSBkZWZhdWx0IiwiaXNzIjoiaHR0cHM6Ly9sb2NhbGhvc3Q6OTQ0My9vYXV0aDIvdG9rZW4iLCJrZXl0eXBlIjoiUFJPRFVDVElPTiIsImV4cCI6MTYyNzU0NTU1NiwiaWF0IjoxNTk2MDA5NTU2LCJqdGkiOiIyN2ZkMWY4Ny01ZTI1LTQ1NjktYTJkYi04MDA3MTFlZTJjZWMifQ==.otDREOsUUmXuSbIVII7FR59HAWqtXh6WWCSX6NDylVIFfED3GbLkopo6rwCh2EX6yiP-vGTqX8sB9Zfn784cIfD3jz2hCZqOqNzSUrzamZrWui4hlYC6qt4YviMbR9LNtxxu7uQD7QMbpZQiJ5owslaASWQvFTJgBmss5t7cnurrfkatj5AkzVdKOTGxcZZPX8WrV_Mo2-rLbYMslgb2jCptgvi29VMPo9GlAFecoMsSwywL8sMyf7AJ3y4XW5Uzq7vDGxojDam7jI5W8uLVVolZPDstqqZYzxpPJ2hBFC_OZgWG3LqhUgsYNReDKKeWUIEieK7QPgjetOZ5Geb1mA==
+    ``` 
+
+2.  We can now invoke the API running on the microgateway using cURL as below. Replace "<NODE_IP>" with one of Kubernetes worker Nodes before
+    executing the command.
+
+    ```bash
+    curl -X GET "https://<NODE_IP>:30201/v2/pet/findByStatus?status=available" -H "accept: application/json" -H "Authorization:Bearer $TOKEN" -k
+    ```
+
+Congratulations! You have successfully created your first API, and invoked it via the API Microgateway.
```

Comment:
You have successfully created your first API on the API Microgateway and invoked it.